CMakeLists.txt

```diff
@@ -102,13 +102,13 @@ set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
 set(bullet_GIT_TAG 4319ffd7e9251066d93064f5a8dab12e33dbe5e2)
 set(bullet_IS_PUBLIC TRUE)
 set(bullet_IS_CMAKE_POD TRUE)
-set(cmake_GIT_REPOSITORY https://github.com/RobotLocomotion/cmake.git)
-set(cmake_GIT_TAG 2d8eebdaf3a7e7a7125c1190462bc02fb3a83824)
+set(cmake_GIT_REPOSITORY https://github.com/rdeits/cmake.git)
+set(cmake_GIT_TAG 800fecfa7842f918d96a96606fa61eb99ce4d03f)
 set(cmake_GIT_CLONE_DIR "${PROJECT_SOURCE_DIR}/drake/cmake")
 set(cmake_NO_BUILD TRUE)
 set(cmake_IS_PUBLIC TRUE)
-set(director_GIT_REPOSITORY https://github.com/RobotLocomotion/director.git)
-set(director_GIT_TAG 628669589d589577dbc1f372529bab8a1e75c926)
+set(director_GIT_REPOSITORY https://github.com/rdeits/director.git)
```

Comment:
yup, now that https://github.com/RobotLocomotion/director/pull/167 is merged I'll restore it